packages/cli/src/commands/dao_cmds/apps.js

```diff
@@ -115,7 +116,7 @@ export const handler = async function({
         task: async (ctx, task) => {
           appsWithoutPermissions = (
             await getAllApps(daoAddress, {
-              web3,
+              web3: new Web3(wsProvider || web3.currentProvider),
```

Comment:
Why do we need to initialize a new Web3 object? Shouldn't be the same as line 107 🤔

Reply:
Good question. Because the `ensureWeb3()` function that generates the `Web3` object doesn't use `wsProvider`. I think this has been fixed in `develop` but not yet on `master`